lib/luathread.c

```diff
@@ -187,6 +236,10 @@ static const luaL_Reg luathread_mt[] = {
 	{"__gc", lunatik_deleteobject},
 	{"stop", luathread_stop},
 	{"task", luathread_task},
+        {"allow", luathread_allow_signal},
+        {"send", luathread_send_signal},
+        {"sigpending", luathread_signal_pending},
+
```

Comment:
I would rather prefix everything or nothing to be consistent.. also, I would name the C function the same